src/main/java/org/assertj/core/error/ShouldHaveBinaryContent.java

```diff
@@ -56,17 +56,17 @@ public static ErrorMessageFactory shouldHaveBinaryContent(InputStream actual, Bi
   }
 
   private ShouldHaveBinaryContent(File actual, BinaryDiffResult diff) {
-    super("%nFile:%n  %s%ndoes not have expected binary content at offset %s, expecting:%n  %s%nbut was:%n  %s",
+    super("%nFile:%n  %s%ndoes not have expected binary content at offset %s, expecting actual:%n  %s%nbut was:%n  %s",
```

Comment:
revert the 3 changes in this file, I realize adding `actual` does not improve the error messages in these cases

Reply:
Yes, agreed. Reverted changes for this file.